x/distribution/genesis.go

```diff
@@ -50,7 +52,16 @@ func ExportGenesis(ctx sdk.Context, keeper Keeper) types.GenesisState {
 		return false
 	})
 	pp := keeper.GetPreviousProposerConsAddr(ctx)
-	outstanding := keeper.GetOutstandingRewards(ctx)
+	outstanding := make([]types.ValidatorOutstandingRewardsRecord, 0)
+	keeper.IterateValidatorOutstandingRewards(ctx,
+		func(addr sdk.ValAddress, rewards types.ValidatorOutstandingRewards) (stop bool) {
+			outstanding = append(outstanding, types.ValidatorOutstandingRewardsRecord{
+				ValidatorAddress:   addr,
+				OutstandingRewards: rewards,
+			})
```

Comment:
let's use `NewValidatorOutstandingRewardsRecord`

Reply:
that isn't a function? these are just internal structs for genesis import/export

Reply:
yeah I meant create then use `
NewValidatorOutstandingRewardsRecord` - it's just good practice to use New constructors incase the fields change

Reply:
agree here ^^

Reply:
Honestly I don't, if we want to do this consistently we'd need to add `NewXYZ` functions for *all* of the genesis structs (none have them at the moment), it seems like quite a bit of misdirecting boilerplate.